sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -26,11 +26,14 @@ import AdditionalDetails from "./sections/AdditionalDetails"
 import AdditionalEligibility from "./sections/AdditionalEligibility"
 import LeasingAgent from "./sections/LeasingAgent"
 import AdditionalFees from "./sections/AdditionalFees"
+import Units from "./sections/Units"
+import { stringToBoolean, stringToNumber } from "../../../lib/helpers"
+import { useAmiChartList } from "../../../lib/hooks"
 import BuildingDetails from "./sections/BuildingDetails"
 import ListingIntro from "./sections/ListingIntro"
 import BuildingFeatures from "./sections/BuildingFeatures"
 
-type FormListing = ListingCreate & ListingUpdate
+type FormListing = Listing
```

Comment:
Curious why you made this change?

Reply:
I was running into issues with units and how it's currently setup with listings. I can look into this further.